packages/simcore-sdk/tests/unit/test_node_ports_v2_r_clone.py

```diff
@@ -79,22 +79,38 @@ async def test__config_file(faker: Faker) -> None:
 
 
 async def test__async_command_ok() -> None:
-    await r_clone._async_r_clone_command("ls", "-la")  # noqa: SLF001
+    result = await r_clone._async_r_clone_command("ls", "-la")  # noqa: SLF001
+    assert len(result) > 0
```

Comment:
is this test really useful?

Reply:
I'd say yes, previously it did not output anything, this would have caught it.